internal/cmd/config.go

```diff
@@ -484,6 +488,22 @@ func Configure(v *viper.Viper, _ *pflag.FlagSet) {
 
 	v.SetDefault("cluster::expiry::enabled", true)
 
+	// ingress controller config
+	defaultIngressControllerValues, err := cluster.NewValuesConfig(map[string]interface{}{
+		"traefik": map[string]interface{}{
+			"ssl": map[string]interface{}{
+				"enabled":        true,
+				"generateTLS":    true,
+			},
+		},
+	})
+	if err != nil {
+		// todo (colin): panic??
+		panic(err)
+	}
+	v.SetDefault("cluster::posthook::ingresscontroller::enabled", true)
+	v.SetDefault("cluster::posthook::ingresscontroller::values", defaultIngressControllerValues)
```

Comment:
This should probably become a string as well. Viper does not play well with custom types.